great-idea-website/css/index.css

```
@@ -63,4 +63,74 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+    width: 900px;
+    margin: auto;
```

this margin: auto can create some funky spacing thats hard to fix if your not careful, you should always set the top and bottom margin to zero. 
```suggestion
    margin: 0 auto;
```